university-college-lillebaelt-vancouver.csl

```diff
@@ -270,7 +278,7 @@
   </citation>
   <bibliography et-al-min="7" et-al-use-first="6" second-field-align="flush">
     <layout>
-      <text variable="citation-number" suffix="."/>
+      <text variable="citation-number" suffix=". "/>
```

Comment:
on which reference manager are you testing this? The space here really shouldn't be necessary.